docs/codegen-options.md

```diff
@@ -0,0 +1,77 @@
+| `fieldsWithResolvers`                           | Set(String)                                        | Empty                                         | Fields that require Resolvers should be defined here in format: `TypeName.fieldName` or `TypeName`. |
+| `fieldsWithoutResolvers`                        | Set(String)                                        | Empty                                         | Fields that DO NOT require Resolvers should be defined here in format: `TypeName.fieldName` or `TypeName`. Can be used in conjunction with `generateExtensionFieldsResolvers` option. |
+| `generateParameterizedFieldsResolvers`          | Boolean                                            | True                                          | If true, then generate separate `Resolver` interface for parametrized fields. If false, then add field to the type definition and ignore field parameters. |
+| `generateExtensionFieldsResolvers`              | Boolean                                            | False                                         | Specifies whether all fields in extensions (<code>extend type</code> and <code>extend interface</code>) should be present in Resolver interface instead of the type class itself. |
+| `subscriptionReturnType`                        | String                                             | Empty                                         | Return type for subscription methods. For example: `org.reactivestreams.Publisher`, `io.reactivex.Observable`, etc. |
+| `generateRequests`                              | Boolean                                            | False                                         | Specifies whether client-side classes should be generated for each query, mutation and subscription. This includes: `Request` class (contains input data) and `ResponseProjection` class (contains response fields). |
+| `requestSuffix`                                 | String                                             | Request                                       | Sets the suffix for `Request` classes. |
+| `responseProjectionSuffix`                      | String                                             | ResponseProjection                            | Sets the suffix for `ResponseProjection` classes. |
+| `parametrizedInputSuffix`                       | String                                             | ParametrizedInput                             | Sets the suffix for `ParametrizedInput` classes. |
+| `parentInterfaces`                              | *See [parentInterfaces](#option-parentinterfaces)* | Empty                                         | Block to define parent interfaces for generated interfaces (query / mutation / subscription / type resolver) |
+
+
+### Option `graphqlSchemas`
+
+When exact paths to GraphQL schemas are too cumbersome to provide in the `graphqlSchemaPaths`, use the `<graphqlSchemas></graphqlSchemas>` block.
```

Comment:
This syntax is maven-specific, there used to be a couple differences like this between the 2 plugin docs (maybe it was overridden already because I don't see the differences anymore).

I like that you extracted the options, it avoids duplication, I think it's worth giving up on the syntax details of each build tool. In this case, though, I think we should avoid the XML syntax and maybe mention just "`graphqlSchemas` block" like this, which would sort of make sense for both plugins.

Reply:
That's a good catch. I've fixed it as per your suggestion.
Thanks, @joffrey-bion!